### PR TITLE
Update readme and add another logo with trimmed edges.

### PR DESCRIPTION
**Pull request by madiator**
* Give an overview and explain key features which should help the user understand why Curator is useful and it's not just another prompting library.
* Add another logo which has less padding on the edges. Note that I can't use this directly in README because the file needs to be checked-in and then I have access via raw.githubusercontent.com.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,22 @@
...
+* We will also be releasing high-quality datasets that should move the needle on post-training.
+
+### Key Features
+
+1. **Programmability and Structured Outputs**: Synthetic data generation is lot more than just calling one prompt. It is a sequence of calling LLMs. You can orchestrate complex pipelines of calling LLMs and use structured output to decide on control-flow. Curator treats structured outputs as first class citizens.
+2. **Built-in Performance Optimization**: We often see calling LLMs in loops, or inefficient implementation of multi-threading. We have baked in performance optimizations so that you don't need to worry about those!
+3. **Intelligent Caching and Fault Recovery**: Given LLM calls can add up in cost and time, failures are undesirable but sometimes unavoidable. We cache the LLM requests and responses so that it is easy to recover from a failure. Moreover, when working on a multi-stage pipeline, caching of stages makes it easy to iterate.
+4. **Native HuggingFace Dataset Integration**: Work directly on HuggingFace Dataset objects throughput your pipeline. Your synthetic data is immediately ready for fine-tuning!
```
Comment (CharlieJCJ) [inline README.md:43]: s/throughput/throughout

File: README.md [hunk 2]
```diff
@@ -26,6 +26,22 @@
...
+Bespoke Curator makes it very easy to create high-quality synthetic data at scale, which you can use to finetune models or use for structured data extraction at scale.
+
+Bespoke Curator is an open-source project:
+* That comes with a rich Python based library for generating and curating synthetic data.
+* A Curator Viewer which makes it easy to view the datasets, thus aiding in the dataset creation.
+* We will also be releasing high-quality datasets that should move the needle on post-training.
+
+### Key Features
+
+1. **Programmability and Structured Outputs**: Synthetic data generation is lot more than just calling one prompt. It is a sequence of calling LLMs. You can orchestrate complex pipelines of calling LLMs and use structured output to decide on control-flow. Curator treats structured outputs as first class citizens.
```
Comment (CharlieJCJ) [inline README.md:40]: s/It is a sequence/It involves a sequence

File: README.md (hunk 1 above)
Comment (madiator) [reply README.md:43]: Done

File: README.md (hunk 2 above)
Comment (madiator) [reply README.md:40]: Updated

File: README.md (hunk 1 above)
Comment (CharlieJCJ) [reply README.md:43]: I don't see updates in this PR?

**Review by CharlieJCJ — APPROVED** [review]
LGTM

**Pull request merged**